### PR TITLE
[VEN-2687][VEN-2820]: Omnichain governance on zksync  & opmainnet

### PR DESCRIPTION
**Pull request by GitGuru7**
## Description

Resolves VEN

### CODE REVIEW

File: vips/vip-365/bsctestnet.ts [hunk 1]
```diff
@@ -0,0 +1,173 @@
...
+      {
+        target: ZKSYNCSEPOLIA_ACM,
+        signature: "giveCallPermission(address,string,address)",
+        params: [
+          ZKSYNCSEPOLIA_OMNICHAIN_EXECUTOR_OWNER,
+          "transferBridgeOwnership(address)",
+          zksyncsepolia.NORMAL_TIMELOCK,
+        ],
+        dstChainId: ZKSYNCSEPOLIA_CHAIN_ID,
+      },
```
[the remark this answers is not in the document]
Comment (GitGuru7) [reply vips/vip-365/bsctestnet.ts:60]: Yes, you are right. I believe we should synchronize the permissions between the mainnet and testnet. For the other testnet networks (Arbitrum Sepolia, Sepolia, and opBNB testnet), we can update the permissions in this PR https://github.com/VenusProtocol/vips/pull/395

File: vips/vip-366/bsctestnet.ts [hunk 2]
```diff
@@ -0,0 +1,132 @@
...
+
+Related with the VIP-366, if passed, this VIP will grant permission to Fast-track and Critical timelocks to execute privilege commands on the OmnichainProposalExecutor on Zksync sepolia. Review VIP-365 for more details.
+
+### References
+`,
+    forDescription: "I agree that Venus Protocol should proceed with this proposal",
+    againstDescription: "I do not think that Venus Protocol should proceed with this proposal",
+    abstainDescription: "I am indifferent to whether Venus Protocol proceeds or not",
+  };
+  return makeProposal(
```
[the remark this answers is not in the document]
Comment (GitGuru7) [reply vips/vip-366/bsctestnet.ts:25]: Yes, you are right. I believe we should synchronize the permissions between the mainnet and testnet.

**Review by chechu — COMMENTED** [review]
sepolia commands lgtm

**Comment by GitGuru7** [comment]
Zksync sepolia Multisig tx : https://sepolia.explorer.zksync.io/tx/0x872e184a74963faaac027605cac8f367bf9de719113d94cefdccb1baa9606472#overview
BNB testnet tx:
execute
https://testnet.bscscan.com/tx/0x2d21f9775e8aa3eaeedabe2773c279e87b25a63495d4babdf669bb56a3066e69#eventlog
Zksync sepolia execute tx :
https://sepolia.explorer.zksync.io/tx/0x0e630dd8eed1d90c04aaa9db2f4fdee9fbb2cd557b4da9e47be99077de53214c

**Review by Debugger022 — APPROVED** [review]

File: src/types.ts [hunk 3]
```diff
@@ -7,18 +7,26 @@ export type SUPPORTED_NETWORKS =
...
   | "ethereum"
   | "opbnbtestnet"
   | "opbnbmainnet"
   | "arbitrumsepolia"
-  | "arbitrumone";
+  | "arbitrumone"
+  | "zksyncsepolia"
+  | "zksyncmainnet"
+  | "opsepolia"
+  | "opmainnet";
```
Comment (kkirka) [inline src/types.ts:25]: Can we do `export type REMOTE_NETWORKS = Exclude<SUPPORTED_NETWORKS, "bscmainnet" | "bsctestnet">`?

**Review by kkirka — APPROVED** [review]
Cool! LGTM

File: src/types.ts (hunk 3 above)
Comment (chechu) [reply src/types.ts:25]: @GitGuru7

**Comment by chechu** [comment]
https://app.venus.io/#/governance/proposal/399?chainId=56

File: src/types.ts (hunk 3 above)
Comment (chechu) [reply src/types.ts]: https://github.com/VenusProtocol/vips/pull/384/commits/a35c9c4fe7d62b364cd288f07d074869d7ab0e40

**Pull request merged**